docs/site/home.html

```diff
@@ -80,9 +80,18 @@ <h3 class="block-title">What is it?</h3>
       </div>
 
       <div class="quote-box--supplementary">
-        <h4>Retrieve Datastore Entities</h4>
-
-        <div hljs language="js">
+        <h4>Example: Retrieve an entity from Cloud Datastore</h4>
+
+        <div class="code-sample-tabs" ng-init="selected = 'compute engine'">
+          <a ng-click="selected = 'anywhere'"
+             ng-class="{selected: selected == 'anywhere'}"
+             >Run anywhere</a>
```

Comment:
Cool - tweaking to:

1. Tabs will be "Run in Compute Engine" and "Run elsewhere"
1. Order of tabs will be "compute engine", then "elsewhere"

Important to note that you can use the "elsewhere" one anywhere (ie, you can be explicit in compute engine about which project and key you want).